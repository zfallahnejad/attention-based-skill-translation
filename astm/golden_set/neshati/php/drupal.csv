830680,drupal
863380,drupal
127549,drupal
46149,drupal
560329,drupal
247353,drupal
784877,drupal
73673,drupal
147937,drupal
573222,drupal
19411,drupal
1902010,drupal
727439,drupal
967168,drupal
711852,drupal
476,drupal
