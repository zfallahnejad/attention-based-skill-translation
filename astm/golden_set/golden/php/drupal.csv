830680,141,55,0.7193877551020408
863380,90,86,0.5113636363636364
127549,63,71,0.4701492537313433
46149,32,24,0.5714285714285714
560329,25,18,0.5813953488372093
247353,22,31,0.41509433962264153
784877,19,9,0.6785714285714286
147937,16,14,0.5333333333333333
73673,16,21,0.43243243243243246
573222,12,15,0.4444444444444444
19411,10,14,0.4166666666666667
1902010,8,11,0.42105263157894735
711852,7,7,0.5
727439,7,7,0.5
476,7,2,0.7777777777777778
967168,7,10,0.4117647058823529
